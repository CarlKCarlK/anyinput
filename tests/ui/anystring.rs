use anyinput::anyinput;

#[anyinput]
fn full_string(s: AnyString<AnyPath>) {
    println!("{}", s);
}

fn main() {}
