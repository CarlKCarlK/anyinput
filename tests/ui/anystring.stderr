error: AnyString should not have a generic parameter, so 'AnyString', not 'AnyString<_>'.
 --> tests/ui/anystring.rs:4:19
  |
4 | fn full_string(s: AnyString<AnyPath>) {
  |                   ^^^^^^^^^^^^^^^^^^
